### ISSUE TITLE
Server Issues

### ISSUE THREAD

**Issue by wardialerMachine** [plnkr/feedback#495]
Stopped allowing me to save, so I logged out.

1. Attempted to log back in via Google, didn't allow me to login.
2. Logged out of Google and attempted again... No good.
3. Created a GitHub account and attempted to login through that... no good.

I love this site, but it seems broken at the moment. I'll try again later!

**_Thank you thank you thank you!!!_**

**Comment by wardialerMachine** [comment]
Tried a bunch of different things, seems to let me in on Microsoft Edge, but the preview doesn't work.
Primarily using Firefox.

**Comment by wardialerMachine** [comment]
Wow, never-mind! It just decided to start working again!

**Comment by wardialerMachine** [comment]
I figured it out... it's a client-side issue... Windows 10, 64-bit... login via Firefox 71...

So, if you have a dual monitor setup, and you are using Firefox on monitor 2... You login with the Google OAUTH thingy, it pops up a new window on the PRIMARY MONITOR (not you guys' fault) and then when I login... the only thing the parent Firefox window gets back is that the window got closed, the login session/token doesn't get passed to Plunker's login panel somehow!

**Solution**: Move Firefox to the monitor 1, then login that way... The popup window can send the login session/token to the parent Firefox window! Strange, but I get it. Security is a priority!
